packages/system-data/src/fields/index.ts

```diff
@@ -29,9 +29,16 @@ import versionFields from './versions.yaml';
 import webhookFields from './webhooks.yaml';
 
 import type { FieldMeta } from '../types.js';
+import { isSystemCollection } from '../collections/index.js';
 
 export const systemFieldRows: FieldMeta[] = [];
 
+export function isSystemField(collection: string, field: string): boolean {
+	if (!isSystemCollection(collection)) return false;
```

Comment:
Isn't this check redundant?

Reply:
How is it redundant?

Reply:
Yes, it's fine the way it is, I just want to be very picky here as we don't actually know that this will be that much faster overall. Theoretically both are the same time complexity group as we're looping over a finite list of things.

Reply:
I know it's probably still not a lot of a difference but the string comparison `if (!collection.startsWith('directus_')) return false;` early return prevents looping over any list if its not a system collection (of course there are edge cases where custom tables start with `directus_`)